thrimshim/thrimshim/main.py

```diff
@@ -184,6 +184,7 @@ def get_row(ident):
 @authenticate
 def update_row(ident, editor=None):
 	new_row = flask.request.json
+        override_changes = 'override_changes' in new_row
```

Comment:
spaces used for indentation here, replace with tabs
```suggestion
	override_changes = 'override_changes' in new_row
```